Readme.md

```diff
@@ -1,37 +1,97 @@
-Then, you need to prepare a cross-compile environment for android. For this purpose, you need to use the
-`/platforms/android/env-android.sh` shell script. Please read the Readme.md in the same directory on how to use that script.
+You may need to install the following packages in your system
 
-*Notes on building webrtc with Android*    
-Building of webrtc is supported only on Linux. This is a limitation of Google's webrtc/chromium build system.  
-Although webrtc comes with its own copy of the NDK, we are going to use the standard one for building the MEGAchat code, and let webrtc build with its own NDK. Both compilers are binary compatible. Forcing webrtc to build with an external NDK will not work. For some operations, like assembly code transformations, a host compiler is used, which is the clang version that comes with webrtc. To use an external NDK, we would need to specify explicitly specify the `--sysroot` path of the external NDK, which also gets passed to the clang host compiler, causing errors. 
+ - `autoconf`  
+ - `automake`  
+ - `libtool`  
+ - `python`  
+ - `g++`  
+ - `qt`
 
-* iOS  
-Building of webrtc and MEGAchat is supported only on MacOS. XCode is required.
-Then, you need to prepare a cross-compile environment for android. For this purpose, you need to use the `/platforms/ios/env-ios.sh` shell script. Please read the Readme.md in the same directory no how to use that script.
+## Chromium build system ##
 
-* Windows
-Microsoft Visual Studio 2015 or later (only C++ compiler), and the Cygwin environment are required. You need to setup the start shortcut/batfile of the Cygwin shell to run the vcvars32.bat with the cmd.exe shell inside the cygwin shell, in order to add the compiler and system library paths to the Cygwin shell. Building of MEGAchat, webrtc and dependencies will be done under the Cygwin shell.
+You may need to get the Chromium build system to get Webrtc (in case `fetch` command fails, you will need to execute this step). Create a directory to get it (chomium_build_dir).
+
+ - `cd chomium_build_dir`  
+ - `git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git`  
+ - `export PATH="$PATH:/chomium_build_dir"`  
+
+## Download WebRTC: ##
+
+Create a directory to download Webrtc (webrtc_dir) and add it to $PATH
+
+ - `cd webrtc_dir`  
+ - `fetch --nohooks webrtc`  
+ - `gclient sync`  
+ - `cd ./src`  
+ - `git checkout c1a58bae4196651d2f7af183be1878bb00d45a57`  
+ - `gclient sync`  
+
+## Get MEGAchat code ##
+
+Checkout the MEGAchat repository:    
+
+ - `git clone --recursive https://github.com/meganz/MEGAchat.git`    
+
+The Megachat sources are at `MEGAchat_repository_dir/src`
+
+*NOTE the `--recursive` switch - the repository contains git submodules that need to be checked out as well.
+
+## Get SDK code ##
+
+Change directory to the root of the MEGAchat checkout  
+
+ - `cd ./third-party/`  
+ - `git clone https://github.com/meganz/sdk.git`    
+ - `mv sdk mega`    
```

Comment:
Suggestion: do `git clone https://github.com/meganz/sdk.git mega`  and remove the `mv`

Reply:
Done